openlibrary/macros/BookPreview.html

```diff
@@ -5,7 +5,7 @@
     <a class="cta-btn cta-btn--preview"
       data-iframe-src="https://archive.org/details/$ocaid?view=theater&wrapper=false"
       data-iframe-link="https://archive.org/details/$ocaid"
-      $:analytics_attr('Preview') href="#bookPreview">$_('Preview') <span>$_('Only')</span></a>
+      $:analytics_attr('Preview') href="#bookPreview">$:_('Preview <span>Only</span>')</a>
```

Comment:
Very weird! Yeah no reason for that to be in a span :P 
```suggestion
      $:analytics_attr('Preview') href="#bookPreview">$_('Preview Only')</a>
```